app/kuma-cp/cmd/cmd_suite_integration_test.go

```diff
@@ -0,0 +1,52 @@
+// +build k8s,integration
+
+package cmd_test
+
+import (
+	"testing"
+
+	"github.com/Kong/kuma/tools/test/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestIntegrationCmd(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Integration CMD Suite")
+}
+
+var _ = Describe("Test K8s deployment with `kumactl install control-plane`", func() {
+
+	It("Deploy on Single K8s cluster and verify the Kuma CP REST API is accessible", func() {
+		t := framework.NewK8sTest(1, "", framework.Silent)
```

Comment:
We could do functional args, so all of those args are optional.
https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis